### PR TITLE
add -Aincap.incremental=true. Solves issue #80

### PR DESCRIPTION
**Pull request by stephanenicolas**
Solves issue #80

### CODE REVIEW

File: ap-client/src/main/java/org/gradle/incap/impl/ProcessorWorkFlowImpl.java [hunk 1]
```diff
@@ -25,14 +26,27 @@ public void init(ProcessingEnvironment processingEnv) {
...
+      File mappingFileFolder = new File(mappingFileFolderPath);
+      mappingFileWriter = new Version1MappingFileWriter(mappingFileFolder);
+      inputTypeFinder = new InputTypeFinder();
+    }
   }
 
   @Override
   public IncrementalFiler createIncrementalFiler(Filer filer) {
-    return new IncrementalFiler(filer, mappingFileWriter, stateGraph, inputTypeFinder);
+    if(isIncremental) {
```
Comment (wangwh03) [inline ap-client/src/main/java/org/gradle/incap/impl/ProcessorWorkFlowImpl.java]: space after if

File: ap-client/src/main/java/org/gradle/incap/impl/ProcessorWorkFlowImpl.java [hunk 2]
```diff
@@ -25,14 +26,27 @@ public void init(ProcessingEnvironment processingEnv) {
...
+      return new IncrementalFiler(filer, mappingFileWriter, stateGraph, inputTypeFinder);
+    }else {
```
Comment (wangwh03) [inline ap-client/src/main/java/org/gradle/incap/impl/ProcessorWorkFlowImpl.java]: space after }

**Comment by wangwh03** [comment]
Looks good, just two space issues, feel free to merge once added. Thanks!

**Comment by stephanenicolas** [comment]
TODO:
* [ ] update the specs docs to mention the change and how the flag works + the `getFiler()` method

**Pull request merged**